index.html

```diff
@@ -75,12 +75,16 @@ <h2>This is our challenge to you.</h2>
                             <section id="schedule" class="main special">
                                 <div class="spotlight">
                                     <div class="content">
+										<!--
                                         <header class="major">
                                             <h2>Schedule: TBD</h2>
                                         </header>
+										-->
                                         <h3><b>Location:</b> <strong>The William Pitt Union, 3959 Fifth Ave., Pittsburgh, PA 15213</strong></h3>
                                         <p></p>
-										<!--
+										<header class="major">
```

Comment:
The indentation here is way off

Reply:
I am completely unsure as to why whenever I upload to Github my indentations keep getting thrown wayyyyyyy off. I can fix this up.